GVFS/GVFS.Common/ProcessHelper.cs

```diff
@@ -62,9 +62,9 @@ public static bool IsDevelopmentVersion()
             return currentVersion.Major == 0;
         }
 
-        public static string WhereDirectory(string processName)
+        public static string GetProgramLocation(string programLocaterCommand, string processName)

```

Comment:
Since we're not going with the `ILocator` component idea (just yet), we might just want to replace `string programLocatorCommand` with `GVFSPlatform.Instance.Constants.ProgramLocatorCommand` since this method is parsing output which is specific to the locator command, and also in practice this method is _always_ going to be called with `GVFS.Platform.Instance.Constants.ProgramLocatorCommand`.

Reply:
The problem is that this `ProcessHelper` class is included in the Hooks projects, which do not have access to `GVFSPlatform` - so it has to be passed in from callers who do (always) have access to `GVFSPlatform`